coalib/output/dbus/BuildDbusService.py

```diff
@@ -1,7 +1,7 @@
 from distutils.core import Command
 from distutils.errors import DistutilsOptionError
 
-from coalib.misc import Constants
+from coalib import BUS_NAME
```

Comment:
shouldn't this be ``from coalib.__init__ import BUS_NAME`` ? or it works like this too?